docs/releasenotes/archive/service/year2017.md

```diff
@@ -6,3 +6,460 @@ description: This is an archive of 2017 Sumo Logic service release notes.
 ---
 
 This is an archive of the 2017 Sumo Logic Service Release Notes. The current Service Release Notes are [here](/docs/releasenotes/service).
+
+---
+## December 20, 2017 (Apps) 
+
+New Beta App - The Sumo Logic App for Amazon SNS is now available. It is a unified logs and metrics (ULM) App that provides insights into the operations and utilization of your SNS service. The preconfigured dashboards help you monitor the key metrics by application, platform, region, and topic name, view the SNS events for activities, and help you plan the capacity of your SNS service.
+
+---
+## December 5, 2017 (Dashboards) 
+
+Enhancement - You can now duplicate dashboards from the dashboard tab. Choose a name for the duplicate and save it to a folder.
+
+---
+## December 5, 2017Live Tail 
+
+Enhancement - Duplicate option now available for Live Tail sessions.
+Bug Fix - Rename option no longer disabled for Live Tail when maximum tab limit reached.
+
+---
+## December 5, 2017 (Metrics) 
+
+Enhancement - New error messages for pct operator when conditions are invalid, such as “Percentile argument greater than 100”.
+Enhancement - Duplicate metrics query option now available.
+Enhancement - Math expressions such as min, max, abs, and round are now case insensitive to match the log search experience.
+Bug Fix - Host Metrics ingest now works with the Setup Wizard.
+Bug Fix - Negative values for log and sqr in queries now result in error messages because they are not valid values for these operators.
+
+---
+## December 5, 2017 (Search) 
+
+Bug Fix - Exporting a search experience improved.
+Bug Fix - To save on naming issues, duplicate imported searches now forced to have a different name than the existing search.
```

Comment:
Needs to be a bulleted list

Reply:
It's done as a bulleted list in DocHub, but it's not needed as they are the information of the release that day, they are not sub-points.

Reply:
Ok! So here you'd just need to put line breaks between each. Otherwise they'll all run together as a giant paragraph.